docs/en/integrations/publish-integration.asciidoc

```diff
@@ -0,0 +1,17 @@
+= Publish an integration
+
+maybe remove this page for now?
```

Comment:
I suggest to write here a short paragraph on how to publish a package in the elastic/integrations repository, I mean - what should developer do to see it in the public registry.